README.md

```diff
@@ -10,150 +10,148 @@
+3. (Optional) **Enable Private Repositories:** Allow the plugin to receive notifications from private repositories by setting this value to `true`.
+4. (**Enterprise Only**) **Enterprise Base URL** and **Enterprise Upload URL**: Set these values to your GitHub Enterprise URLs, e.g. `https://github.example.com`. The Base and Upload URLs are often the same. When enabled, existing users must reconnect their accounts to gain access to private repositories. Affected users will be notified by the plugin once private repositories are enabled.
+5. Hit **Save**.
+6. Go to **System Console > Plugins > Management** and click **Enable** to enable the GitHub plugin.
 
-You're all set! To test it, run the `/github connect` slash command to connect your Mattermost account with GitHub.
+You're all set!
 
-### Configuration Notes in High Availability Mode
+## Using the Plugin
 
-If you are running Mattermost v5.11 or earlier in [High Availability mode](https://docs.mattermost.com/deployment/cluster.html), please review the following:
+Once configuration is complete, run the `/github connect` slash command from any channel within Mattermost to connect your Mattermost account with GitHub.
 
-1. To install the plugin, [use these documented steps](https://docs.mattermost.com/administration/plugins.html#plugin-uploads-in-high-availability-mode)
-2. Then, modify the config.json [using the standard doc steps](https://docs.mattermost.com/deployment/cluster.html#updating-configuration-changes-while-operating-continuously) to the following
+## Onboarding Your Users
 
-```
-"PluginSettings": {
-    ...
-    "Plugins": {
-        "github": {
-            "encryptionkey": "<your encryption key, from step 5 above>",
-            "githuboauthclientid": "<your oauth client id, from step 2 above>",
-            "githuboauthclientsecret": "<your oauth client secret, from step 2 above>",
-            "githuborg": "<your github org>",
-            "webhooksecret": "<your webhook secret, from step 3 above>"
-        },
-    ...
-    "PluginStates": {
-        ...
-        "github": {
-            "Enable": true
-        },
-        ...
-    }
-},
-```
+When you’ve tested the plugin and confirmed it’s working, notify your team so they can connect their GitHub account to Mattermost and get started. Copy and paste the text below, edit it to suit your requirements, and send it out.
 
-## Development
+> Hi team, 
 
-This plugin contains both a server and web app portion.
+> We've set up the Mattermost GitHub plugin, so you can get notifications from GitHub in Mattermost. To get started, run the `/github connect` slash command from any channel within Mattermost to connect your Mattermost account with GitHub. Then, take a look at the [slash commands](https://github.com/mattermost/mattermost-plugin-github#slash-commands) section for details about how to use the plugin.
```

Comment:
That link does't work right now. Can you confirm that it will after the merge?

Reply:
I've adjusted it.